barretenberg_wasm/src/lib.rs

```diff
@@ -21,6 +21,10 @@ pub use common::crs;
 use std::cell::Cell;
 use wasmer::{imports, Function, Instance, Memory, MemoryType, Module, Store, Value};
 
+const FIELD_BYTES: usize = 32;
+const SIG_BYTES: usize = 64;
```

Comment:
small nit: do we want to call this SCHNOR_SIG_BYTES since not all signatures are the same, or is this implied from the context?

Reply:
comments on what these are would be good